src/features/skeleton/Skeleton.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+
+const Skeleton = ({ mode }) => {
+  return (
+    <div>
+      {mode === "singleCard" && (
+        <div className="skeleton_card_container">
+          <div className="skeleton_card"></div>
+        </div>
+      )}
+      {mode === "groupCards" && (
```

Comment:
En vez de repetir las cards tantas veces el skeleton podría recibir una prop que sea un número y hacer un for. Esta lógica la pondría antes del return y no directamente en el JSX. Dependiendo del número va a devolver x cantidad de divs skeleton_card

Reply:
Solucionado